integration/inspektor-gadget/integration_test.go

```diff
@@ -909,6 +911,57 @@ func TestOpensnoop(t *testing.T) {
 	RunCommands(commands, t)
 }
 
+func TestNetworkGraph(t *testing.T) {
+	ns := GenerateTestNamespaceName("test-networkgraph")
+
+	t.Parallel()
+
+	networkGraphCmd := &Command{
+		Name:         "StartNetworkGadget",
+		Cmd:          fmt.Sprintf("$KUBECTL_GADGET trace network -n %s -o json", ns),
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntry := &networkTypes.Event{
+				Event: eventtypes.Event{
+					Type: eventtypes.NORMAL,
+					CommonData: eventtypes.CommonData{
+						Namespace: ns,
+						Pod:       "test-pod",
+					},
+				},
+				PktType:    "OUTGOING",
+				Proto:      "tcp",
+				Port:       443,
+				RemoteKind: "other",
+			}
+
+			normalize := func(e *networkTypes.Event) {
+				e.Node = ""
+				e.Container = ""
+				e.IP = ""
+				e.PodHostIP = ""
+				e.PodIP = ""
+				e.PodOwner = ""
+				e.PodLabels = nil
```

Comment:
You can also verify the `PodLabels` as the pod we create using `BusyboxPodRepeatCommand` to generate the events has the label `run=test-pod`:

https://github.com/inspektor-gadget/inspektor-gadget/blob/b586002292b6ac79295d9657496d30ffb9c635a4/integration/command.go#L499-L500